AndelaOSP/andela-societies-frontend
# [#158489323] Success Ops view of the Verify Activities page

#### What does this PR do?
Displays the `Success Ops` view of the Verify Activities page.
#### Description of Task to be completed?
Have the following working: 
- Verify activities page to have a linear view
- Checkbox in activity cards to enable selecting an activity
- Select all functionality
- Approve selected functionality that approves only selected activities
#### How should this be manually tested?
`git pull` this branch, `git checkout` to it. In your terminal `yarn start`. You'll needd to login as a `success ops` member, then navigate to `Verify activities` page via the side bar.
#### Any background context you want to provide?
I branched off `ft-156998268-add-activity-approval-buttons`in order to work on this task hence the large number of files changed.
#### What are the relevant pivotal tracker stories?
[158489323](https://www.pivotaltracker.com/story/show/158489323)
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-06-27 at 12 49 10" src="https://user-images.githubusercontent.com/29278184/41967044-5cd58d14-7a09-11e8-9946-d4fcc14640ba.png">
<img width="1440" alt="screen shot 2018-06-27 at 12 59 48" src="https://user-images.githubusercontent.com/29278184/41967307-fc1ffe22-7a09-11e8-84b9-5b9929ef6a4a.png">
